common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -920,3 +920,26 @@ def test_video_with_multiple_transcripts_index_dictionary(self):
             },
             "content_type": "Video"
         })
+
+    def test_video_with_no_transcripts_index_dictionary(self):
+        """
+        Test index dictionary of a video module with
+        no transcripts uploaded by a user.
+        """
+        xml_data_transcripts = '''
+            <video display_name="Test Video"
+                   youtube="1.0:p2Q6BrNhdh8,0.75:izygArpw-Qo,1.25:1EeWXzPdhSA,1.5:rABDYkeK0x8"
+                   show_captions="false"
+                   download_track="false"
+                   start_time="00:00:01"
+                   download_video="false"
+                   end_time="00:01:00">
+              <source src="http://www.example.com/source.mp4"/>
+              <track src="http://www.example.com/track"/>
+              <handout src="http://www.example.com/handout"/>
+            </video>
+        '''
+
+        descriptor = instantiate_descriptor(data=xml_data_transcripts)
+        translations = descriptor.available_translations(descriptor.get_transcripts_info(), False);
```

Comment:
Nit: For readability, pass `False` as a keyword argument, so it's clear for what argument it's being sent: `verify_assets=False`.